embeds/docs-dearborn-michigan/index.html

```diff
@@ -0,0 +1,31 @@
V46.9l-.5.04c-.5.06-.9.4-.9 1.05" opacity=".8"/><path d="M104.07 50h-3.7v-.62l.72-.34v-3.66c0-.7-.3-.95-.9-.95-.4 0-.7.08-1 .2v4.4l.7.35v.6h-3.6v-.6l.7-.34v-4.57l-.8-.35v-.6l2.8-.43h.2v.8c.6-.6 1.5-.9 2.4-.9 1.1 0 1.7.5 1.7 1.8V49l.7.35v.6zm-10.1-1.23c.87 0 1.35-.1 1.82-.23v.65c-.4.4-1.4.9-2.6.9-2.3 0-3.5-1.3-3.5-3.6 0-2.2 1.3-3.5 3.3-3.5 2 0 3 .9 3 3.7h-4c.1 1.3.7 1.9 1.9 1.9zM89.82 50h-3.54v-.62l.6-.34v-3.66c0-.7-.24-.95-.84-.95-.23 0-.5.06-.75.17v4.44l.6.34v.6h-3.5v-.6l.6-.34v-3.66c0-.7-.3-.95-.9-.95-.3 0-.5.06-.8.2v4.4l.6.35v.6h-3.5v-.6l.7-.34v-4.57l-.8-.35v-.6l2.8-.43h.1v.8h.1c.6-.6 1.2-.9 2.1-.9.8 0 1.3.2 1.5.8h.1c.6-.6 1.24-.9 2.2-.9 1.1 0 1.7.5 1.7 1.8V49l.7.35v.6zm28.9-.97l.93.34V50h-3.88v-.63l.7-.34v-2.3l-1.55-.63v2.94l.72.34v.5c-.22.12-.73.28-1.27.28-.7 0-1.33-.23-1.43-.95h-.06c-.3.6-1.1 1-1.94 1-1.2 0-1.94-.7-1.94-1.9 0-1.3.9-1.6 2.38-1.9l1.3-.2v-1l-1.85-.7-.32 1h-1V44l-.7-.3v.83h-1.4V48c0 .65.2 1 .9 1 .3 0 .6-.04.8-.06v.88c-.4.2-1.1.4-1.8.4-1.3 0-1.92-.6-1.92-2.03v-3.8H104v-1l1.03-.2V42L79.46 31.5l-1.52 1.05-.02 16.5.4.2v.58l-.4.04-.02 16.18 1.5 1.18 41.26-16.94v-1.9l-1.92-.8V49zM69 45.38l-.1-.33-1.73-.7c-.32.37-.52 1.06-.52 2.15 0 1.6.83 2.27 1.93 2.27.63 0 1.3-.1 1.64-.2v.27l.83-.35v-2.6l-1.34-.6H69zm-8.47 4.8c-2.05 0-3.46-1.27-3.46-3.56 0-2.28 1.4-3.53 3.46-3.53 2.05 0 3.46 1.2 3.46 3.5s-1.5 3.5-3.5 3.5zm-3.64-.34l-2.7.3H54v-.7h-.06c-.28.36-.86.73-1.7.73-1.23 0-2.7-.78-2.7-3.37 0-2.7 1.5-3.7 3.3-3.7.36 0 .8.1 1.08.3v-1.66l-.75-.34v-.6l2.8-.46h.18v8.52l.7.4v.58zm-16.3 9.23c.5-1.1.8-2.37.8-3.78 0-4.3-2.4-6.3-7.2-6.3h-7.7c-.9 0-1.6-.6-1.6-1.4 0-.6.4-1.2.9-1.6 1.2.3 2.2.4 3.6.4 5.6 0 9.4-2.5 9.4-7.2v-.6c-.1-2.5-1.2-4.3-3.1-5.6 5.6 2.4 9.6 8 9.6 14.5 0 4.4-1.9 8.4-4.8 11.3zm-11 2.05c-4.1 0-5.6-1.44-5.6-3 0-1.1.6-2.55 2.6-2.55h5.8c1.9 0 3 1.04 3 2.53 0 1.7-1.3 3.02-6 3.02zm-6.2-6.83c-2.5.5-5.2 1.5-5.2 4.3v.2c-2.8-2.9-4.6-6.8-4.6-11.1 0-6.4 3.8-11.9 9.2-14.4-1.9 1.3-2.9 3.2-2.9 5.6 0 2.8 1.4 4.9 3.6 6.1l.1.1c-1.3.8-3.8 2.8-3.8 5.1 0 1.7 1.1 3.3 3.4 3.8zm6-20.2c1.6 0 2.9.6 2.9 5 0 4.3-1.4 4.7-2.9 4.7-1.6 0-2.9-.4-2.9-4.8s1.3-5 2.8-5zm34.8 12.5c0-1.3.5-2.3 1.3-2.9L14 22.4 11.9 24v46.3l2.1 1.44 53-21.7c-1.8-.28-2.75-1.6-2.75-3.5z" opacity=".8"/><path d="M51.94 46.58c0 1.66.42 2.38 1.3 2.38.35 0 .54-.06.7-.12v-4.56c-.17-.13-.37-.24-.76-.24-.7 0-1.24.64-1.24 2.54m8.6-2.58c-.75 0-1.12.4-1.12 2.62 0 2.23.37 2.6 1.12 2.6.75 0 1.1-.37 1.1-2.6 0-2.22-.35-2.6-1.1-2.6" opacity=".8"/><path d="M60.54 49.23c.7 0 1.06-.35 1.1-2.3l-2.2-.92c-.02.2-.02.4-.02.7 0 2.3.37 2.6 1.12 2.6m-8.6-2.6c0 1.7.42 2.4 1.3 2.4.35 0 .54 0 .7-.1v-4.6c-.17-.1-.37-.2-.76-.2-.7 0-1.24.7-1.24 2.6" opacity=".8"/><path d="M67.82 50.16c-1.7 0-2.77-.77-3.23-2.03l-.8-.3c-.4 1.52-1.7 2.34-3.3 2.34-2.1 0-3.5-1.26-3.5-3.55 0-.55.1-1.05.2-1.48l-1.2-.47v4.2l.7.4v.57l-2.7.3v-.7h-.1c-.3.36-.9.73-1.7.73-1.3 0-2.7-.78-2.7-3.37 0-2.5 1.3-3.52 2.8-3.67l-6.8-2.8-1.6 1.1c.9 1.96 1.33 4.1 1.33 6.38 0 2.3-.5 4.5-1.38 6.5v22.4l1.6 1.3 43.3-17.8v-2L69 50c-.34.1-.75.2-1.18.2" opacity=".8"/></g></svg>
```

Comment:
you [need to remove](http://stuntbox.com/blog/2013/06/bulletproof-inline-svg/) the `xmlns` attribute here